project/test_framework/.idea/CiviWikiTest.iml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Lets add `.idea` to `.gitignore`, so we don't keep that folder under revision control.